src/main/java/dk/cngroup/lentils/controller/ClientController.java

```diff
@@ -5,10 +5,12 @@
 import dk.cngroup.lentils.entity.Hint;
 import dk.cngroup.lentils.entity.Team;
 import dk.cngroup.lentils.entity.formEntity.Codeword;
+import dk.cngroup.lentils.exception.NotOneHourBeforeOpeningTimeException;
```

Comment:
Nepojmenoval bych to OneHourBefore... jakmile se nekdo rozhodne zmenit cas, tak nazev vyjimky bude matouci.